js2-mode.el

```diff
@@ -56,16 +56,13 @@
 
 ;;   (add-hook 'js-mode-hook 'js2-minor-mode)
 
+;; If you use Emacs 27 and desire JSX support added in that release,
```

Comment:
You might want to merge this paragraph with the existing one before the code sample. Otherwise, it reads a bit weird.

Especially with the existing paragraph below which suggests to "also ... hook it", but actually refers to `js2-mode`.

Reply:
Certainly, I made that change.